R/expr__expr.R

```diff
@@ -1373,7 +1373,7 @@ Expr_rechunk = "use_extendr_wrapper"
 #' @aliases Expr_cumsum
 #' @name Expr_cumsum
 #' @details
-#' Dtypes in {Int8, UInt8, Int16, UInt16} are cast to
+#' Dtypes in `{Int8, UInt8, Int16, UInt16}` are cast to
```

Comment:
Thanks, but I'm not sure if it's appropriate to express these as "code" here.

How about

```suggestion
#' Dtypes in Int8, UInt8, Int16, UInt16 are cast to
```

Reply:
makes sense, how about something like this commit for all the braces?
https://github.com/pola-rs/r-polars/pull/424/commits/45eb76638518c2fc23f6f16b07ffc9e0ff2d9847

Reply:
I think it is good.

In this style it should be possible to check the arguments using the `match.arg` function.
Like this:
https://github.com/apache/arrow/blob/c5bce96ba626ad2558e10acddf8ef19ea06940e8/r/R/dataset-write.R#L125-L136

Reply:
We don't need match.arg as that is already taken care of in the conversion on rust-side and prints a very nice error, mentioning all valid options. If using match.arg we also have to catch raised errors and forward them

Reply:
Makes sense, thanks.

Reply:
The match.arg - like conversion

https://github.com/pola-rs/r-polars/blob/326c370abc88f23e0e7aa611363da98b72617784/src/rust/src/rdatatype.rs#L299